draft-ietf-ice-rfc5245bis.xml

```diff
@@ -776,56 +761,56 @@ it to send a check.
 consequence of the receipt of a connectivity check from the peer.
 </t>
 
-<t hangText="VALID LIST:"> An ordered set of candidate pairs for a
+<t hangText="Valid List:"> An ordered set of candidate pairs for a
 media stream that have been validated by a successful STUN
 transaction.
 </t>
 
-<t hangText="Check List Set:"> An ordered list of CHECK LISTs.
+<t hangText="Check List Set:"> The ordered list of all check lists.
+The order is determined by each ICE usage.
 </t>
 
-<t hangText="Full:"> An ICE implementation that performs the complete
-set of functionality defined by this specification.
+
+<t hangText="Full Implementation:"> An ICE implementation that
+performs the complete set of functionality defined by this
+specification.
 </t>
 
-<t hangText="Lite:"> An ICE implementation that omits certain
+<t hangText="Lite Implementation:"> An ICE implementation that omits certain
 functions, implementing only as much as is necessary for a peer
 implementation that is full to gain the benefits of ICE. Lite
 implementations do not maintain any of the state machines and do not
 generate connectivity checks.
 </t>
 
-<t hangText="Controlling Agent:"> The ICE agent that is responsible
-for selecting the final choice of candidate pairs and signaling them
-through STUN. In any session, one agent is always controlling. The
+<t hangText="Controlling Agent:"> The ICE agent that nominates a
+candidate pair.  In any session, one agent is always controlling. The
 other is the controlled agent.
 </t>
 
-<t hangText="Controlled Agent:"> An ICE agent that waits for the
-controlling agent to select the final choice of candidate pairs.
+<t hangText="Controlled Agent:"> The ICE agent that waits for the 
+controlling agent to nominate a candidate pair.
 </t>
 
-<t hangText="Nomination, Regular Nomination:"> The process of picking a valid
-candidate pair for media traffic by validating the pair with one
-STUN request, and then picking it by sending a second STUN request
-with a flag indicating its nomination.
+<t hangText="Nomination, Regular Nomination:"> The process of the
+controlling agent indicating to the controlled agent which candidate
+pair the controlled agent should use to send media.
 </t>
 
 <t hangText="Nominated:"> If a valid candidate pair has its nominated
 flag set, it means that it may be selected by ICE for sending and
 receiving media.
 </t>
 
-<t hangText="Selected Pair, Selected Candidate:"> The candidate pair
-selected by ICE for sending and receiving media is called the selected
-pair, and each of its candidates is called the selected candidate.
-Before a  pair has been selected, any valid candidate pair
-can be used for sending and receiving media (only one candidate pair
-at any given time).
```

Comment:
My think is that any valid candidate pair can be used for receiving *any time*, not just before a pair has been selected.  Why would you throw away media that you receive on a valid candidate pair just because it comes out of order with the candidate pair selection (or for any other reason)?

But it's true that's not just editorial. Sorry about that.  I can make it a separate PR/issue/thread if you'd like.